azurerm/resource_arm_cognitive_account.go

```diff
@@ -100,6 +100,18 @@ func resourceArmCognitiveAccount() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"key1": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+
+			"key2": {
```

Comment:
whilst I appreciate this is named `key2` in the API - could we name this `secondary_access_key` for consistency with the other resources?